packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -5,8 +5,9 @@
  *
  * @prop --calcite-tile-background-color: Specifies the background color of the component.
  * @prop --calcite-tile-border-color: Specifies the border color of the component.
- * @prop --calcite-tile-description-text-color: Specifies the description text color of the component.
```

Comment:
Wouldn't we still want this? There are two different colors used for the heading / description UI ... I'd expect a css property to individually control each. cc @ashetland @SkyeSeitz to confirm.

Reply:
@alisonailea 's suggestion is simply to replace `--calcite-tile-description-text-color` with `--calcite-tile-text-color` which effectively serves as like the "body" default text color, and because it is applied to the `.container` div instead of the `.description` div, it will set a color for the whole content of the tile, including icons and other content.

Reply:
Description and heading are different colors currently and will remain different colors in future Tile enhancements.

Reply:
Since there isn’t a default content slot, I’d expect the css props to match the property name for each. 

Other content is slotted in. This is why I still think the “icon prop” would have its own css property. 

As a user without knowledge of the component, setting the icon to the description color is confusing (or, having description / icon be text-color but then also having a “named” text color property for heading).